client/wfnews-war/src/main/angular/src/app/components/report-of-fire/review-page/rof-review-page.component.ts

```diff
@@ -287,4 +290,45 @@ export class RoFReviewPage extends RoFPage implements AfterViewInit{
       return { success: false, message: 'An error occurred while submitting the report' };
     }
   }
+
+submitRof(){
+  const rofResource = {
+    fullName: this.nullEmptyStrings(this.reportOfFire.fullName),
+    phoneNumber: this.nullEmptyStrings(this.reportOfFire.phoneNumber),
+    consentToCall: equalsIgnoreCase(this.reportOfFire.consentToCall, "Yes") ? true : false,
+    estimatedDistance: this.reportOfFire.estimatedDistance,
+    fireLocation: this.reportOfFire.fireLocation,
+    fireSize: this.nullEmptyStrings(this.reportOfFire.fireSize),
+    rateOfSpread: equalsIgnoreCase(this.reportOfFire.rateOfSpread, "Yes") ? "Fast" : equalsIgnoreCase(this.reportOfFire.rateOfSpread, "No") ? "Slow" : "Unknown",
+    burning: this.reportOfFire.burning,
+    smokeColor: this.reportOfFire.smokeColor,
+    weather: this.reportOfFire.weather,
+    assetsAtRisk: this.reportOfFire.assetsAtRisk,
+    signsOfResponse: this.reportOfFire.signsOfResponse,
+    otherInfo: this.reportOfFire.otherInfo
+  }
+
+  try {   
+    const response = this.reportOfFireService.saveReportOfFire(rofResource, this.reportOfFire.image1, 
```

Comment:
I'm doing the null check here in the service: https://github.com/ssylver93/nr-bcws-wfnews/blob/feature/WFNEWS-1419/client/wfnews-war/src/main/angular/src/app/services/report-of-fire-service.ts#L20 I think that should be ok? Otherwise I think I would need to create 4 different functions to take in just the body, the body and 1 image, the body and 2 images, the body and 3 images?